### ISSUE TITLE
Lazy loading vertical nav: Content loaded confirmation is not communicated. 

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4625]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 4.1.3 Status messages

**Test Environment:**
- Test URL: https://clarity.design/documentation/vertical-nav/basic-structure/project-pokemon
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 19.3.1

**Steps to reproduce:**
1. Open the test URL.
2. Go to Lazy loading nav.
3. Activate the lazy loading content and observe screen reader's spoken feedback.

**Expected result:**
There should be some way to notify screen reader user that the content is loaded.

**Actual result:**
Only content loading status is announced. Once the content is loaded there is no confirmation to the screen reader user.

**Proposed solution:**
1. One way to do is via placing the focus on the first element from the loaded content, which has proper semantics to tell the user that the content is from loaded content. For e.g. Clicking on Menu, content is loaded, menu items are displayed and focus is placed on the first menu item. Screen reader announces the item label and notifies user about the number of menu items. This way user understands that the content from the loaded Menu.
2. Other way is to provide confirmation message that the content is loaded. This option should be used if the first option is not available.

**Comment by gkhadK** [comment]
Similar issue is observed for loading buttons. On screen changes are not communicated clearly after pressing the button. Sighted users can see the loading activity and understand that the button will be unavailable during the loading time. But similar info is not available to the screen reader user. Only partial info available to the screen reader user with some workaround is; the unavailability of button in loading state. To understand that info, screen reader user has to navigate away and refocus that button.

**Comment by gkhadK** [comment]
Similar issue is observed for indeterminate checkbox - No confirmation provided after reset indeterminate state action is performed by pressing the reset button.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
